doc/source/index.rst

```diff
@@ -26,7 +26,13 @@ Pyorbital has a module for parsing NORAD TLE-files
     >>> tle.inclination
     99.043499999999995
 
-If no path is given pyorbital tries to read the earth observation TLE-files from celestrak.com
+If no path is provided pyorbital first tries to read any local tle files in the
+directory given by the environment variable TLES. If this environment is not
+set Pyorbital will try get the earth observation TLE-files over the internet
+from celestrak.com. Please observe, that it is only in the sace that no
+specific tle file is provided or if the TLES environment variable is not set
+that TLE fetch over the internet is attempted.
```

Comment:
A couple typos and rewordings, but the phrase "environment variable TLES" is confusing. Is that a variable named "TLES" or are those TLEs found in the directory pointed to by the PYORBITAL_CONFIG_PATH? And is it "TLE" or "tle", it seems to be switched around a lot in the docs.

```suggestion
If no path is provided pyorbital first tries to read any local tle files in the
directory given by the environment variable TLES. If this environment is not
set Pyorbital will try get the earth observation TLE files over the internet
from celestrak.com. Note this downloading if no
specific tle file is provided or if the TLES environment variable is not set.
```

Reply:
> A couple typos and rewordings, but the phrase "environment variable TLES" is confusing. Is that a variable named "TLES" or are those TLEs found in the directory pointed to by the PYORBITAL_CONFIG_PATH? And is it "TLE" or "tle", it seems to be switched around a lot in the docs.

Yes, should be "TLE" rather than "tle". And, yes, TLES is an environment variable - the path to where the user keeps her/his local TLE files. Not sure if this env variable could actually be a list of directories. Need to check that. Do you remember @loerum ?

Reply:
Ok then if TLES is an env var it should at least be highlighted with two backticks on either side, but I think you can use this in sphinx to make it extra clear:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-envvar

Reply:
Thanks, that is really nice. It works.

Reply:
Done